lib/ansible/modules/cloud/azure/azure_rm_networkinterface.py

```diff
@@ -173,6 +179,19 @@
             security_group_name: secgroup001
             public_ip_address_name: publicip001
 
+    - name: Create a network with mutilple ip configurations
```

Comment:
Are you also able to update the existing examples so that the single IP configuration uses the `ip_configurations` list instead of the deprecated args.